### PR TITLE
Install and configure rubocop

### PR DESCRIPTION
**Pull request by madwort**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
 
 gemspec
+
+gem 'rubocop', '~> 0.42.0'
```
Comment (meddle0x53) [inline Gemfile]: It should be part of the `reacto.gemspec`, something like `spec.add_development_dependency 'rubocop', '~> 0.42.0'`

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+  Exclude:
+    - 'lib/reacto/operations/wrap.rb'
+    - 'spec/reacto/trackable/split_labeled_spec.rb'
+
+# Offense count: 13
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: final_newline, final_blank_line
+Style/TrailingBlankLines:
+  Exclude:
```
Comment (meddle0x53) [inline .rubocop_todo.yml:464]: Wow, lot's of them!

File: .rubocop_todo.yml [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+# SupportedStyles: use_perl_names, use_english_names
+Style/SpecialGlobalVars:
+  Exclude:
+    - 'reacto.gemspec'
+
+# Offense count: 14
+# Cop supports --auto-correct.
+# Configuration parameters: IgnoredMethods.
+# IgnoredMethods: respond_to, define_method
+Style/SymbolProc:
```
Comment (meddle0x53) [inline .rubocop_todo.yml:434]: That's a good one

File: .rubocop_todo.yml [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+  Exclude:
+    - 'lib/reacto/trackable.rb'
+    - 'spec/reacto/trackable/inject_spec.rb'
+
+# Offense count: 7
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, EnforcedStyleForEmptyBraces, SpaceBeforeBlockParameters.
+# SupportedStyles: space, no_space
+Style/SpaceInsideBlockBraces:
+  Exclude:
```
Comment (meddle0x53) [inline .rubocop_todo.yml:411]: Again, I'm surprised...

**Comment by meddle0x53** [comment]
Very good idea, @madwort, thanks for the support!
I have only one comment about where to place the dependency and it is good to be merged.

File: .rubocop_todo.yml [hunk 5]
```diff
@@ -0,0 +1,493 @@
...
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Style/EmptyLinesAroundMethodBody:
+  Exclude:
+    - 'lib/reacto/operations/cycle.rb'
+
+# Offense count: 7
+# Cop supports --auto-correct.
+Style/EvenOdd:
```
Comment (meddle0x53) [inline .rubocop_todo.yml:222]: I knew about this one, even started updating the specs, but then forgot...

**Comment by meddle0x53** [comment]
Will have to add some CI integration after that.

File: Gemfile (hunk 1 above)
Comment (madwort) [reply Gemfile]: 👍 done

**Pull request merged**